Running "Subcase test"...
12CHECK(false) at subcases.cpp:8 failed!
    in subcase a
CHECK(false) at subcases.cpp:9 failed!
    message: msg1
    in subcase a
3CHECK(false) at subcases.cpp:13 failed!
    in subcase a
    in subcase b
CHECK(false) at subcases.cpp:14 failed!
    message: msg2
    in subcase a
    in subcase b
56end
===== Tests passed: 0/1 =====
